corehq/apps/data_interfaces/tests/test_case_deduplication.py

```diff
@@ -49,10 +47,7 @@ def setUp(self):
 
     def _prime_es_index(self, cases):
         for case in cases:
-            case_search_adapter.index(
-                transform_case_for_elasticsearch(case.to_json()),
-                refresh=True
-            )
+            case_search_adapter.index(case, refresh=True)
```

Comment:
nit: could use `bulk_index`

Reply:
Also other places in this test

Reply:
https://github.com/dimagi/commcare-hq/pull/32660/commits/ca5fdb0bb3d8ba3c6f19041f8c8097ea75156bda